audio_service/lib/audio_service.dart

```diff
@@ -2973,10 +3019,10 @@ class AudioServiceBackground {
   /// broadcasting such a position change, the [updateTime] specifies the time
   /// of that change, allowing clients to project the realtime value of the
   /// position as `position + (DateTime.now() - updateTime)`. As a convenience,
-  /// this calculation is provided by [PlaybackState.currentPosition].
+  /// this calculation is provided by [PlaybackState.position].
   ///
   /// The playback [speed] is given as a double where 1.0 means normal speed.
-  static Future<void> setState({
+  static void setState({
```

Comment:
This is a breaking change. It should be a backwards compatibility layer and therefore match the type signature of the old version of this method.

Reply:
do you plan to keep this forever?

Reply:
No, it will be phased out, that is why it is deprecated (via the enclosing class). It may have been better for me to explicitly annotate each member as deprecated rather than just the class as that would allow me to give a pointer to each new corresponding API.

Reply:
i see